### ISSUE TITLE
Amend ASN.1 definition of CABFOrganizationIdentifier to require at least 1 character for state/province value

### ISSUE THREAD

**Issue by CBonnell** [cabforum/servercert#452]
The current ASN.1 definition of the `CABFOrganizationIdentifier` extension value is as follows:

```asn1
CABFOrganizationIdentifier ::= SEQUENCE {
registrationSchemeIdentifier PrintableString (SIZE(3)),
registrationCountry PrintableString (SIZE(2)),
registrationStateOrProvince [0] IMPLICIT PrintableString
(SIZE(0..128)) OPTIONAL,
registrationReference UTF8String
}
```

This definition technically allows for the inclusion of a 0-character `registrationStateOrProvince` value. This is not needed, as the `jurisdictionStateOrProvinceName` will always contain at least 1 character, if present.

This definition should be modified to specify the size constraint for the `registrationStateOrProvince` element to be 1 to 128 characters. 

In other words:
```asn1
CABFOrganizationIdentifier ::= SEQUENCE {
registrationSchemeIdentifier PrintableString (SIZE(3)),
registrationCountry PrintableString (SIZE(2)),
registrationStateOrProvince [0] IMPLICIT PrintableString
(SIZE(1..128)) OPTIONAL,
registrationReference UTF8String
}
```

**Comment by aarongable** [comment]
Notably, usage of `SIZE(1..` vastly outnumbers useage of `SIZE(0..` in RFC 5280, so think change would bring the definition more in line with how most things are defined in x509.

**Comment by dzacharo** [comment]
Addressed in SC095
